services/web-app/src/lib/config.ts

```diff
@@ -0,0 +1,14 @@
+export const clientId = process.env.GITHUB_ID;
+export const clientSecret = process.env.GITHUB_SECRET;
+
+if (!clientId || !clientSecret) {
+  throw new Error('Missing required environment variables GITHUB_ID and/or GITHUB_SECRET.');
+}
+
+export const awsAccessKeyId = process.env.AWS_ACCESS_KEY;
```

Comment:
This feels weird to me that we are using access keys and secret keys for these things. There might be better ways to do this, might be good to speak with @mattzcarey about it